templates/vue/src/components/Lightbox/media/MultiContentMedia/index.vue

```diff
@@ -329,6 +332,7 @@ button[disabled] {
 }
 
 .nested-media-title {
+  color: #111;
```

Comment:
Again, not sure if we need this on a child element - can we keep it on the parent elements instead? Is there an overwrite somewhere that should be removed instead?

Reply:
yep, the media wrapper takes care of this rule... just checked